content/help/ops/security/debugging-authorization/index.md

```diff
@@ -0,0 +1,258 @@
+      for the target service.
+
+1. For example, you might see something similar to the following:
+
+    {{< text plain >}}
+    2018-07-26T22:25:41.009838Z debug rbac building filter config for {sleep.foo.svc.cluster.local map[app:sleep pod-template-hash:3326367878] map[destination.name:sleep destination.namespace:foo destination.user:default]}
+    2018-07-26T22:25:41.009915Z info  rbac no service role in namespace foo
+    2018-07-26T22:25:41.009957Z info  rbac no service role binding in namespace foo
+    2018-07-26T22:25:41.010000Z debug rbac generated filter config: { }
+    2018-07-26T22:25:41.010114Z info  rbac built filter config for sleep.foo.svc.cluster.local
+    2018-07-26T22:25:41.182400Z debug rbac building filter config for {productpage.default.svc.cluster.local map[pod-template-hash:2600844901 version:v1 app:productpage] map[destination.name:productpage destination.namespace:default destination.user:bookinfo-productpage]}
+    2018-07-26T22:25:41.183131Z debug rbac checking role app2-grpc-viewer
+    2018-07-26T22:25:41.183214Z debug rbac role skipped for no AccessRule matched
+    2018-07-26T22:25:41.183255Z debug rbac checking role productpage-viewer
+    2018-07-26T22:25:41.183281Z debug rbac matched AccessRule[0]
+    2018-07-26T22:25:41.183390Z debug rbac generated filter config: {policies:<key:"productpage-viewer" value:<permissions:<and_rules:<rules:<or_rules:<rules:<header:<name:":method" exact_match:"GET" > > > > > > principals:<and_ids:<ids:<any:true > > > > >  }
+    2018-07-26T22:25:41.184407Z info  rbac built filter config for productpage.default.svc.cluster.local
+    {{< /text >}}
+
+    It means Pilot have generated:
+
+    * An empty config for `sleep.foo.svc.cluster.local` as there is no authorization policies matched
+      and Istio denies all requests sent to this service by default.
+
+    * An config for `productpage.default.svc.cluster.local` and Istio will allow anyone to access it
+      with GET method.
+
+## Ensure Pilot Distributes Policies to Proxies Correctly
+
+Pilot distributes the authorization policies to proxies. The following steps help you ensure Pilot
+is working as expected:
+
+> Note: The command used in this section assumes you have deployed [Bookinfo application](/docs/examples/bookinfo/),
+otherwise you should replace `"-l app=productpage"` with your actual pod.
+
+1. Run the following command to get the proxy configuration dump for the `productpage` service:
+
+    {{< text bash >}}
+    $ kubectl exec  $(kubectl get pods -l app=productpage -o jsonpath='{.items[0].metadata.name}') -c istio-proxy -- curl localhost:15000/config_dump -s
+    {{< /text >}}
+
+1. Check the log and verify:
+
+    * The log includes an `envoy.filters.http.rbac` filter to enforce the authorization policy
+      on each incoming request.
+    * Istio updates the filter accordingly after you update your authorization policy.
+
+1. For example, you might see something similar to the following:
```

Comment:
Replace this line with the following text:
1. The following output means the proxy of `productpage` has enabled the `envoy.filters.http.rbac` filter with rules that allows anyone to access it via `GET` method. The `shadow_rules` are not used and you can ignored them safely.